apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go

```diff
@@ -169,3 +183,13 @@ func (i *ClowdJobInvocation) SetObjectMeta(o metav1.Object, opts ...omfunc) {
 		opt(o)
 	}
 }
+
+func (i *ClowdJobInvocation) GetInvokedJobs(ctx context.Context, c client.Client) (batchv1.JobList, error) {
+
+	jobs := batchv1.JobList{}
+	if err := c.List(ctx, &jobs, client.InNamespace(i.ObjectMeta.Namespace)); err != nil {
+		return jobs, err
```

Comment:
If you make the return a pointer to a JobList, you can return nil here, which I think makes sense from a readability PoV and could make things a littler more optimised too